tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -1477,6 +1579,120 @@ feature! {
     {
         subscriber_impl_body! {}
     }
+
+
+    impl<C, S> Subscribe<C> for Vec<S>
+    where
+        S: Subscribe<C>,
+        C: Collect,
+    {
+
+        fn on_subscribe(&mut self, collector: &mut C) {
+            for s in self {
+                s.on_subscribe(collector);
+            }
+        }
+
+        fn register_callsite(&self, metadata: &'static Metadata<'static>) -> Interest {
+            // Return highest level of interest.
+            let mut interest = Interest::never();
+            for s in self {
+                let new_interest = s.register_callsite(metadata);
+                if (interest.is_sometimes() && new_interest.is_always())
+                    || (interest.is_never() && !new_interest.is_never())
+                {
+                    interest = new_interest;
+                }
+            }
+
+            interest
+        }
+
+        fn enabled(&self, metadata: &Metadata<'_>, ctx: Context<'_, C>) -> bool {
+            // XXX(eliza): does this need to clear the filter state when it
+            // returns false, or will `Layered` always take care of that for us?
```

Comment:
would be good to figure out!

Reply:
oh, i forgot to reply that it doesn't, `Layered` takes care of that --- this was a reminder to myself to write tests for this. it can be removed:
```suggestion
```